pkg/kubectl/cmd/create_role.go

```diff
@@ -267,6 +267,19 @@ func (c *CreateRoleOptions) validateResource() error {
 			return err
 		}
 	}
+
+	if len(c.ResourceNames) > 0 {
+		illegalVerbs := []string{}
+		for _, verb := range c.Verbs {
+			switch verb {
+			case "list", "watch", "create", "deletecollection":
```

Comment:
@CaoShuFeng these verbs are illegal. the reason is at https://github.com/kubernetes/kubernetes.github.io/pull/4149#discussion_r123147422